src/App/App.tsx

```diff
@@ -71,14 +74,47 @@ library.add(faExternalLinkSquareAlt);
 toast.configure({
   autoClose: TOAST_AUTO_CLOSE_DELAY /** defines how long a toast remains visible on screen */,
 });
-initGoogleAnalytics();
+
+const initializeOptions = {
+  testMode: GA_ENV_TEST === GA_ENV,
+};
+
+/** Initialize google analytics */
+initGoogleAnalytics(GA_CODE, initializeOptions);
```

Comment:
@kelvin-muchiri just a nit-pick: please name-space this `initializeOptions` to be more specific to google analytics 🙏